packages/block-editor/src/components/warning/test/index.js

```diff
@@ -18,7 +18,9 @@ describe( 'Warning', () => {
 
 	it( 'should show primary actions', () => {
 		render(
-			<Warning actions={ <button>Click me</button> }>Message</Warning>
```

Comment:
I'm not sure why we allowed this behavior. The prop is used and documented as an array.